packages/core/test/core/plan/dependency.test.ts

```diff
@@ -351,8 +354,10 @@ describe('dependency changers', () => {
         ])
         dependencyChanges = [...await addReferencesDependency(inputChanges, new Map())]
       })
-      it('should not add dependency', () => {
-        expect(dependencyChanges).toHaveLength(0)
+      it('should add dependency', () => {
```

Comment:
nit: worth mentioning that the dependency is to the element and not the value
```suggestion
      it('should add dependency to the element that contains the target', () => {
```